doc/src/tutorial/matrices.rst

```diff
@@ -459,7 +459,7 @@ by injecting a custom zero test with warnings enabled.
     ...         result = None
     ...
     ...     # Warnings if evaluated into None
-    ...     if result == None:
+    ...     if result is None:
```

Comment:
How about writing
if not result:
instead of 
if result is None:

Reply:
So, basically, no. We figured out that return type is `Optional[bool]`. Thus, `is None` is an adequate comparison here.